Doc/c-api/complex.rst

```diff
@@ -118,6 +118,12 @@ Complex Numbers as Python Objects
 
    This subtype of :c:type:`PyObject` represents a Python complex number object.
 
+   .. c:member:: Py_complex cval
+
+      The complex number value, using the C :c:type:`Py_complex` representation.
+
+      .. deprecated-removed:: next 3.17
```

Comment:
Maybe we should recommend PyComplex_AsCComplex() here?

Reply:
And `PyComplex_FromCComplex` too?

Reply:
If it's a subtype instance they've just allocated, it might even work.